src/lib/adapters/msteams.js

```diff
@@ -89,11 +89,4 @@ MSTeamsAdapter.prototype.normalizeCommand = function (command) {
   return command;
 }
 
-MSTeamsAdapter.prototype.normalizeAddressee = function(msg) {
```

Comment:
What happened to this `MSTeamsApater` code block, why it's been removed?

Reply:
Good question, I left this out of the PR description.

The MS Teams adapter [inherits from `DefaultAdapter`](https://github.com/StackStorm/hubot-stackstorm/blob/3a14cdd8ed73fb7f2c3647df215e02035065eebd/src/lib/adapters/msteams.js#L28), and this code for `normalizeAddressee` in the MS Teams adapter is the exact same as the [`normalizeAddressee` function from `DefaultAdapter`](https://github.com/StackStorm/hubot-stackstorm/blob/3a14cdd8ed73fb7f2c3647df215e02035065eebd/src/lib/adapters/default.js#L82). Overriding the function was an oversight in #186.

I've updated the PR description to include this. I figured the PR tagline was sufficiently ambiguous to include this change as well.